docker-compose.yml

```
@@ -48,3 +49,16 @@ services:
     restart: always
     ports:
       - 6379
+
+  minio:
+    container_name: ieducar-minio
+    image: minio/minio
+    volumes:
+      - ./data1:/data
+    restart: always
+    ports:
+      - 9001:9000
+    environment:
+      MINIO_ACCESS_KEY: minio
+      MINIO_SECRET_KEY: minio123
```

Para facilitar a identificação.

```suggestion
      MINIO_ACCESS_KEY: minio-key
      MINIO_SECRET_KEY: minio-secret
```